altos-rust/src/shell.rs

```diff
@@ -215,6 +225,21 @@ pub fn shell(_args: &mut Args) {
                     let hms = uptime();
                     println!("{:02}:{:02}:{:02}", hms.0, hms.1, hms.2);
                 },
+                Command::Rocket => {
+                    let timer:isize = if words.len() > 0 {
```

Comment:
Type can be elided, so `let timer = ...` instead of `let timer: isize = ...`
And type should be separated by a space between the colon, so `let timer: isize` instead of `let timer:isize`